tests/python/PythonPackageInitBearTest.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import os
+
+from queue import Queue
+
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+from bears.python.PythonPackageInitBear import PythonPackageInitBear
+
+
+class PythonPackageInitBearTest(unittest.TestCase):
+
+    def setUp(self):
+        
+        os.mkdir('with-init')
+
+
+        open('with-init/__init__.py', 'a').close()
+
+        os.mkdir('ignored-no-init')
+        open('ignored-no-init/ignored.py', 'a').close()
+
+        os.mkdir('not-ignored-no-init')
+        open('not-ignored-no-init/test-file.py', 'a').close()
+
+        self.queue = Queue()
+        self.file_dict = {}
+        self.section = Section('PythonPackageInit')
+        self.section.add_or_create_setting(Setting('ignored_dirs', 'ignored-no-init'))
+
+        self.uut = PythonPackageInitBear(self.file_dict, self.section,
+                                         self.queue)
+
+    def tearDown(self):
+        os.remove('with-init/__init__.py')
+        os.remove('ignored-no-init/ignored.py')
+        os.remove('not-ignored-no-init/test-file.py')
+        os.rmdir('with-init')
+        os.rmdir('ignored-no-init')
+        os.rmdir('not-ignored-no-init')
```

Comment:
Also maybe you should move all this code to the setup block before you make thing to make sure everything is gone? I'd recommend testing for these files to make it fail-proof.